netbox/netbox/models/mixins.py

```diff
@@ -55,7 +55,7 @@ def clean(self):
 class DistanceMixin(models.Model):
     distance = models.DecimalField(
         verbose_name=_('distance'),
-        max_digits=8,
+        max_digits=16,
```

Comment:
I don't follow why this needs to be increased. `distance` is the "native" unit (e.g. miles, kilometers, etc.); six digits with a precision of two decimal points seems sufficient. Only the absolute distance (in meters) might need to be increased to support a high native value (e.g. 9,999 miles).

Reply:
My expectation here was that this also needed to accommodate entering something in much smaller units. For instance 100,000 miles is 528,000,000 feet, which would overflow the 10^6 limit that currently exists. I can see why it wouldn't make sense to enter such a large magnitude of a smaller unit, when larger units are available.